src/main/java/io/debezium/connector/yugabytedb/connection/pgproto/YbProtoMessageDecoder.java

```diff
@@ -24,6 +24,7 @@
 import io.debezium.connector.yugabytedb.proto.PgProto.Op;
 import io.debezium.connector.yugabytedb.proto.PgProto.RowMessage;
 import io.debezium.util.Collect;
+import org.yb.cdc.CdcService;
```

Comment:
Seems like this is the only change in this file. If so then it can be removed if it's not being used.